holidays/index.js

```diff
@@ -0,0 +1,19 @@
+export {default as at} from './at.js';
```

Comment:
I just found that there is also a [YAML plugin](https://github.com/rollup/rollup-plugin-yaml) :heart: So I would prefer YAML. But I can also convert that later.